ckanext/unaids/react/components/BulkFileUploaderComponent/src/App.js

```diff
@@ -49,13 +49,13 @@ export default function App({ lfsServer, maxResourceSize, orgId, datasetId, defa
         axios.post(
             '/api/3/action/resource_create',
             {
-                package_id: datasetId,
+                package_id: datasetName,
```

Comment:
It looks like id/name can be used interchangeably without issues.
I've tested this and it works.